scully/startBackgroundServer.ts

```diff
@@ -0,0 +1,28 @@
+import {spawn} from 'child_process';
+import {existsSync} from 'fs-extra';
+import {join} from 'path';
+import {ScullyConfig} from './utils/interfacesandenums';
+import {logError, log, green} from './utils/log';
+
+export function startBackgroundServer(scullyConfig: ScullyConfig) {
+  const binary = ['/dist/scully/scully', '/node_modules/.bin/scully', '/node_modules/@scullyio/scully/scully']
+    .map(p => join(scullyConfig.homeFolder, p + '.js'))
+    .find(p => existsSync(p));
+
+  if (!binary) {
+    logError('Could not find scully binaries');
+    process.exit(15);
+    return;
+  }
+  console.log('starting', binary);
```

Comment:
Should this console.log be `log` instead? Or removed?

Reply:
I think this is a left-over, I think it should be removed.